libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -206,6 +220,32 @@ describe('ModalWrapperComponent', () => {
     });
   });
 
+  // TODO: I need a little help here.
+  // describe('with default themed footer', () => {
+  //   beforeEach(() => {
+  //     spectator = createComponent({
+  //       props: {
+  //         config: {
+  //           title: 'Modal with footer',
+  //           component: DefaultThemedFooterEmbeddedComponent,
+  //         },
+  //       },
+  //     });
+  //   });
+
+  //   afterEach(() => {
+  //     // Ensure any observers are destroyed:
+  //     spectator.component.ngOnDestroy();
+  //   });
+
+  //   it('should default to white background on embedded footer', () => {
+  //     const kirbyModalFooter = spectator.element.querySelector(':scope > kirby-modal-footer');
+  //     expect(kirbyModalFooter).toHaveComputedStyle({
+  //       'background-color': getColor('white'),
+  //     });
+  //   });
+  // });
+
```

Comment:
I'm missing something here...I get this error:
Expected [background-color] of KIRBY-MODAL-FOOTER 'rgba(0, 0, 0, 0)' to equal 'rgb(255, 255, 255)' (white)